app/views/pages/start.html.erb

```diff
@@ -43,10 +43,14 @@
       Have your National Insurance number ready, as we’ll ask for this.
     </p>
 
-    <%= govuk_start_button(
-      text: "Start now",
-      href: "#",
-      classes: 'govuk-!-margin-top-2 govuk-!-margin-bottom-9') %>
+    <%= form_with model: @trn_request do %>
+      <button class="govuk-button govuk-!-margin-top-2 govuk-!-margin-bottom-9 govuk-button--start" data-module="govuk-button">
+        Start now
+        <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
+          <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+        </svg>
+      </button>
+    <% end %>
```

Comment:
Interesting, did we have to use the inline HTML content for a `govuk_start_button` because that component is technically a link?

Also we couldn't use `f.govuk_submit` I guess because that's not a start button.

We will revisit this and turn this back to a link later on, so should be fine.

Reply:
Why does this need to be a form here?

Reply:
@fofr it's temporary. Because we're building the application backwards, we have to take users directly to the confirmation page. We need to create an entry in the database to show users. The idiomatic way to do this is using a POST request. It's possible to do it via a GET, but this has unintended consequences.